db/seeds.rb

```diff
@@ -1,7 +1,53 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+    require 'faker'
```

Comment:
Not sure how to force Faker to use longer names. I'm a little surprised that Faker would produce such a short name. Maybe there is something else going on? Anyway, assuming that the problem is with Faker sometimes generating too-short names, I put a loop in the seeds.db that keeps generating names until it gets one with more than 5 characters.